sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2022-05-25)
 
```

Comment:
Could we also add [these contents](https://github.com/ArthurMa1978/azure-sdk-for-net/blame/19d18500b6887881b6e1091d14de7f0213b7a969/sdk/containerapps/Azure.ResourceManager.Applications.Containers/CHANGELOG.md#L5-L11) here?

Reply:
No problem